api/bazel/repository_locations.bzl

```diff
@@ -16,9 +16,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "protoc-gen-validate (PGV)",
         project_desc = "protoc plugin to generate polyglot message validators",
         project_url = "https://github.com/envoyproxy/protoc-gen-validate",
```

Comment:
`https://github.com/bufbuild/protoc-gen-validate`

Reply:
seems like there are a few more refs beyond the deps hookup - ive opened a ticket  #23424